packages/dashboard/src/components/account-button-toggle/accounts-button-toggle.scss

```diff
@@ -0,0 +1,32 @@
+.button-toggle {
+    &__container {
+        width: 20.8rem;
+        height: 4.8rem;
```

Comment:
Ideally we'd not fix width and let the content decide, also would it be possible to create a "container" in components? We're using a toggle container in P2P too, so we'd be able to re-use it there.